crates/consensus/consensus/Cargo.toml

```diff
@@ -17,6 +17,7 @@ reth-primitives.workspace = true
 # misc
 auto_impl.workspace = true
 thiserror-no-std = {workspace = true, default-features = false }
+thiserror.workspace = true
```

Comment:
this defeats the purpose of the thiserror-no-std crate, we only need thiserror-no-std here

Reply:
Oh ok, I thought that due to the feature gates on the imports that `thiserror-no-std` would get used when `std` is not allowed and `thiserror` would get used when `std` is active.